site/config/api/config.json

```diff
@@ -1,28 +0,0 @@
-{
```

Comment:
This `site.config/api` directory needs to remain in some capacity because it's used to autogenerate our API reference doc. If moved, then the Makefile target `generate-api-docs` should reference the new location of the config.

Reply:
I think it'll be somewhat odd to have an otherwise empty folder left for the API reference document: I'll examine the Makefile target and think of a higher-level folder to keep it in.